Sources/Purchasing/Purchases/PurchaseParams.swift

```diff
@@ -89,6 +92,23 @@ import Foundation
             return self
         }
 
+        #if !ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION
+        /**
+         * Sets a win-back offer for the purchase.
+         * - Parameter winBackOffer: The ``WinBackOffer`` to apply to the purchase.
+         *
+         * Fetch a winBackOffer to use with this function with ``Purchases/eligibleWinBackOffers(forProduct:)``
+         * or ``Purchases/eligibleWinBackOffers(forProduct:completion)``.
+         *
+         * Availability: iOS 18.0+, macOS 15.0+, tvOS 18.0+, watchOS 11.0+, visionOS 2.0+
+         */
+        @available(iOS 18.0, macOS 15.0, tvOS 18.0, watchOS 11.0, visionOS 2.0, *)
+        internal func with(winBackOffer: WinBackOffer) -> Self {
```

Comment:
We've removed the availability check from `WinBackOffer` so maybe we can remove it here as well?

Reply:
I think it's important so that folks aren't using it in iOS <18. I had to remove the check from the WinBackOffer itself for some other reason, but this keeps people from being able to use it when it's not available, so I think it should probably stay